signer/src/testing/wsts.rs

```diff
@@ -39,6 +41,48 @@ pub struct SignerInfo {
     pub signer_public_keys: BTreeSet<PublicKey>,
 }
 
+/// Trait for selecting the coordinator's private key from a list of items which
+/// contain keypairs.
+pub trait SelectCoordinatorPrivateKey {
+    /// Select the coordinator's private key based on the given list of `SignerInfo`.
```

Comment:
nit: mentioning `SignerInfo` here seems unexpected

Reply:
yeah, missed to update it, [4c577f5](https://github.com/stacks-sbtc/sbtc/pull/1727/commits/4c577f589c91ae212f328ce24cd99f2b7ae28dd3)